src/styles/search.css

```diff
@@ -0,0 +1,4 @@
+.search {
+        margin: auto;
+        padding: 20px 0px;
+}
```

Comment:
See comment about margin above. Also you need to make the formatting uniform to the other files to use 2 spaces a with the other files
```
.search {
  margin: 0 auto;
  padding: 20px 0px;
}
```